### DISCUSSION TITLE
Alpha masking not working in SceneViewer AR

### DISCUSSION THREAD

**Discussion by Aloalo** [google/model-viewer#3634] [Q&A]
### Description

SceneViewer AR seems to ignore alpha masking. In the attached asset, the leaves are alpha masked, but the parts that should be discarded appear as a solid white plane in AR. This happens on Samsung Galaxy S20+, Note 10 Ultra and Note 20 Ultra.

### Version
- model-viewer: v1.7.2
- model-viewer: v1.9.0

### Browser Affected
- [x] Chrome, version: 96.0.4664.104
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS
- [x] Android 11
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [x] SceneViewer
- [ ] QuickLook

[HassAvocado.zip](https://github.com/google/model-viewer/files/7757168/HassAvocado.zip)

**Comment by elalish** [comment]
@tpsiaki Is glTF MASK mode supported in Scene Viewer?

**Comment by Sky-XR** [comment]
Checking on this issue. Any update? Thank you!
